util/regexp-assemble/regexp-942170.data

```diff
@@ -1,3 +1,21 @@
+##! This is a data file used to generate a regular expression for a CRS rule. 
+##! The generation of the regular expression happens with the help of 
+##! util/regexp-assemble/regexp-assemble.pl.
+##! The ID of the rule in question is part of the file name of this data file. 
+##! Read more about the format of this data file and the use of the assembly 
+##! script in util/regexp-assemble/README.md.
+##!
+##! Lines starting with `##!` are comments and will be skipped,
+##! empty lines will be ignored completely.
+##! 
+##! Three special comments are at your disposal to influence the assembled expression:
+##! - `##!+`: the flag comment
+##! - `##!^`: the prefix comment
+##! - `##!$`: the suffix comment
+##! Please refer to util/regexp-assemble/README.md for a full explanation.
+
+##!+ i
```

Comment:
Ignore case flag should be changed in the rule file so that it reflects the result of the assembling script:

```
$ diff 942170.old 942170.new
1c1
< (?i:(?:select|;)\s+(?:benchmark|sleep|if)\s*?\(\s*?\(?\s*?\w+)
---
> (?i)(?:select|;)\s+(?:benchmark|sleep|if)\s*?\(\s*?[(]?\s*?\w+
```

Reply:
Done. I also updated the data file as the parenthesis `(` wasn't being escaped but presented as a character class (which doesn't make sense).